lbcpresenter/src/main/kotlin/studio/lunabee/compose/foundation/presenter/LBPresenter.kt

```diff
@@ -43,7 +43,12 @@ import kotlinx.coroutines.flow.stateIn
  */
 typealias LBSimpleReducer<UiState, NavScope, Action> = LBReducer<out UiState, UiState, NavScope, Action, out Action>
 
-abstract class LBPresenter<UiState : PresenterUiState, NavScope : Any, Action> : ViewModel() {
+/**
+ * @property verbose enable verbose logs using kermit logger
+ */
+abstract class LBPresenter<UiState : PresenterUiState, NavScope : Any, Action>(
+    private val verbose: Boolean = false,
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/LunabeeStudio/Lunabee_Compose_Android/blob/cc1f9bf65b7ecd0bd600df5f2b530d4ef1a98440/lbcpresenter/src/main/kotlin/studio/lunabee/compose/foundation/presenter/LBPresenter.kt#L50"></span>Private property <code>verbose</code> is unused.</td>
    </tr>
  </tbody>
</table>